examples/brace-sections.jsgold

```diff
@@ -0,0 +1 @@
+["Hello, World!"  4 42 'phew]
```

Comment:
The missing `{_}` must be https://github.com/msp-strath/Mary/issues/36 ; just recording it here.